src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -295,4 +295,53 @@ public void saveEventInformation(List<Event> events, PreparedQuery results) {
   }
 
 
+  /**This function returns one Event given a query**/
+  
+  public void saveIndividualEvent(Event event,PreparedQuery results) {
+	
+	for (Entity entity : results.asIterable()) {
```

Comment:
Why are there multiple results? Shouldn't there just be one entity that your'e saving?